src/static/style.css

```diff
@@ -0,0 +1,187 @@
+body {
+  margin: 0;
+}
+
+* {
+  font-family: 'Inter', sans-serif;
+}
+
+.status {
+  display: flex;
+  align-items: center;
+}
+
+.status-hide {
+  display: none;
+}
+
+.loader {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  width: 40px;
+  height: 40px;
+}
+
+.loader div {
+  box-sizing: border-box;
+  display: block;
+  position: absolute;
+  width: 10px;
+  height: 10px;
+  margin: 0px;
+  border: 1px solid #fff;
+  border-radius: 50%;
+  animation: loader 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;
+  border-color: #ff4e3d transparent transparent transparent;
+}
+
+.loader div:nth-child(1) {
+  animation-delay: -0.45s;
+}
+
+.loader div:nth-child(2) {
+  animation-delay: -0.3s;
+}
+
+.loader div:nth-child(3) {
+  animation-delay: -0.15s;
+}
+
+@keyframes loader {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  padding: 0 5vw 0 5vw;
+}
+
+.td-container {
+  display: flex;
+  justify-content: center;
+}
+
+.title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: auto;
+  font-weight: bold;
+  font-size: 36px;
+  height: 10vh;
+}
+
+.instructions {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 16px;
+  width: 100%;
+  height: 5vh;
+}
+
+.command {
+  --tw-bg-opacity: 1;
+  --tw-text-opacity: 1;
```

Comment:
what are these?

Reply:
css variables, used here:
<img width="517" alt="Screenshot 2021-12-29 at 4 38 57 PM" src="https://user-images.githubusercontent.com/52193408/147656249-08fffddf-d34f-41e2-817f-589e528c9136.png">

I can hardcode them if needed, since the designs were made by me(not a designer) anyway.

Reply:
What browser support does that limit us to?

Reply:
Only internet explorer doesn't support this kind of variables: https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#browser_compatibility

Of course, internet explorer will be discontinued on June 15, 2022: https://support.microsoft.com/en-us/windows/internet-explorer-downloads-d49e1f0d-571c-9a7b-d97e-be248806ca70

Reply:
Official platform support is unrelated to packages supporting a platform - as long as people use IE, it should be supported by websites, even after Microsoft has stopped.

I'm also concerned though about older versions of Chrome/Firefox/Safari - no browser is actually "evergreen", that's just a marketing lie :-)

Reply:
Haha  that makes sense, I will simply hardcode the variables into the code since in vanilla CSS most of the things are hardcoded anyway.